app/src/helpers/helpers.ts

```diff
@@ -87,3 +87,8 @@ export function getCounterValue(title: string): string {
   const match = itemCountRegex.exec(title);
   return match ? match[1] : undefined;
 }
+
+export function isArgFormatValid(arg: string): boolean {
+  // eslint-disable-next-line
```

Comment:
I think I saw in a failed CI build that this ignore is because [eslint/prefer-regexp-exec](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-regexp-exec.md) wants you to use `RegExp.exec`, correct?

If correct, sorry for the successive changes (I already asked you to change from `String.search` to `String.match`), but I'm going to suggest we do yet another change and go with the RegExp.exec recommended by ESLint. Ignoring the recommendation wouldn't be dramatic since performance is moot here (we're talking about a few meager calls), but switching to the alternative is fine-looking too, so I'd rather have it than an easily-avoided eslint-ignore.

Reply:
I'll make the change!